From e30dc240e7417b83957507f1da6b34c31f18afb5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Dan=20Hor=C3=A1k?= <dan@danny.cz>
Date: Tue, 29 Jun 2010 22:23:50 +0200
Subject: [PATCH] fix --dates option on zfcpdbf

Description: zfcpdbf: Fix --dates option
Symptom:     Running zfcpdbf with the option --dates returns "Unknown
             option: dates".
Problem:     The code expects --date, not the documented --dates option.
Solution:    Change zfcpdbf to expect --dates, as documented in the
             man page and in the usage information.
---
 scripts/zfcpdbf |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/zfcpdbf b/scripts/zfcpdbf
index 3693d71..0b75c43 100755
--- a/scripts/zfcpdbf
+++ b/scripts/zfcpdbf
@@ -139,7 +139,7 @@ GetOptions(
 	'V|verbose'	=> \$OPT_VERBOSE,
 	'h|help'	=> \$OPT_HELP,
 	'v|version'	=> \$OPT_VERSION,
-	'D|date=s'	=> \$OPT_DATE,
+	'D|dates=s'	=> \$OPT_DATE,
 	's|singleline'	=> \$OPT_SINGLELINE
 ) or print_usage();
 
@@ -555,7 +555,7 @@ sub print_san {
 
 #
 # Converts the Unix time to localtime an returns it 
-# depending on the --date option.
+# depending on the --dates option.
 #
 # \param	UNIX Timestamp to convert
 # \return 	Formated Localtime
-- 
1.6.6.1

